specification/TileFormats/Batched3DModel/README.md

```diff
@@ -23,7 +23,7 @@ _Batched 3D Model_ allows offline batching of heterogeneous 3D models, such as d
 
 Per-model properties, such as IDs, enable individual models to be identified and updated at runtime, e.g., show/hide, highlight color, etc. Properties may be used, for example, to query a web service to access metadata, such as passing a building's ID to get its address. Or a property might be referenced on the fly for changing a model's appearance, e.g., changing highlight color based on a property value.
 
-A Batched 3D Model tile is a binary blob in little endian.
+An instance of a Batched 3D Model tile is a binary blob in little endian.
```

Comment:
I'm not really sure if it's necessary to say "an instance of". It sounds more confusing that way.